src/components/Image/Image.jsx

```diff
@@ -1,23 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import style from './style.scss';
+import cn from 'classnames';
+import styles from './style.scss';
+
 import {
   getDataAttrs,
   getDataProps,
 } from '../../dataUtils';
 
+const defaultImage = 'https://cdn.spacetelescope.org/archives/images/screen/s82e5407.jpg';
```

Comment:
do we still want this default image btw? it's a photo of astronauts lol

Reply:
lol, tt's just used as a default prop for src and fallback, so kept it.